runtime/prompty/prompty/utils.py

```diff
@@ -29,7 +29,7 @@ async def load_json_async(file_path, encoding='utf-8'):
     return json.loads(content)
 
 def _find_global_config(prompty_path: Path = Path.cwd()) -> Path:
-    prompty_config = list(Path.cwd().glob("**/prompty.json"))
+    prompty_config = list(prompty_path.glob("**/prompty.json"))
```

Comment:
This *may cause an issue when the `prompty.json` file is declared in a path inside of a parent folder (which in theory it should pick up). I am interested in your thoughts on the behavior here.

Reply:
It looks like it is inconsistent with the async version (which is my bad) - am still curious on your thoughts here.

Reply:
I felt that the config should be related to executing file, but now thinking it better, I would argue that it would be better to follow what other libs do, which I think is:
 1. Search for config file in CWD (just the current dir)
 2. If not found, then use a well known folder. Like $HOME/.prompty/cfg or similar.
 
But this search assuming the prompty file is in a subfolder of CWD seems wrong to me.

Finally, it would be great to see the global config using key/val human readable format like YAML or properties.